docs/installation.rst

```diff
@@ -140,6 +140,24 @@ If set to ``True``, strings will be HTML escaped. By default this is ``False``.
 If set to ``True``, strings will be sanitized by removing any leading '=' character.  This is to prevent execution of
  Excel formulae.  By default this is ``False``.
 
+``IMPORT_EXPORT_FORMATS``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Defines which file formats will be allowed during imports and exports. Defaults
```

Comment:
Please can you clarify that this needs to be defined as a list of class names taken from `base_formats.py`, probably best by providing an example.

Reply:
Done